blockchain/validate.go

```diff
@@ -109,6 +118,17 @@ var (
 	// earlyFinalState is the only value of the final state allowed in a
 	// block header before stake validation height.
 	earlyFinalState = [6]byte{0x00}
+
+	// The following blocks violate DCP0005 as they were submitted with an old
+	// version after the majority of the network had upgraded.  See
+	// isDCP0005Violation for more details.  They are defined as package level
+	// variables to avoid the need to create new instances every time a check is
+	// needed.
+	block413762Hash = newHashFromStr("00000000000000002086ed61f62a546f3bfa8f3e567b003f097efa982008c47b")
```

Comment:
Please use `mustParseHash` instead.  We want it to panic if there is an error parsing a hard-coded value as opposed to ignoring it.  The `newHashFromStr` can be removed.

Reply:
Updated to use `mustParseHash` and removed `newHashFromStr` and removed all references to `newHashFromStr` within the blockchain tests.